examples/java-change-collection-type/index.ts

```diff
@@ -0,0 +1,24 @@
+import { JavaGenerator } from '../../src';
+
+const generator = new JavaGenerator({
+  collectionType: 'List'
+});
+const jsonSchemaDraft7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    email: {
+      type: 'string',
+      format: 'email'
```

Comment:
I can see I did not mention this in the issue. However, we need to change the input so it is rendered as a list rather than a single value. This means that instead of a single `string` type for `email`, we need to change it to be an array of them. Otherwise the example output wont show the correct expected output where arrays are rendered as lists 🙂 

Let me know if you want the answer or ping me if you are having trouble 🙂

Reply:
you mean we change from `type : 'string' ` to `type: Array('string') ` ? Only then the `email` property accepts an array of strings.

Reply:
The input is nothing more than JSON Schema draft 7, [so you can follow their documentation](https://json-schema.org/understanding-json-schema/reference/array.html) 🙂